GOVERNANCE.md

```diff
@@ -0,0 +1,125 @@
+are identified by the WG and their addition as Collaborators is discussed
+during the weekly WG meeting.
+
+**Note**: If you make a significant contribution and are not considered for
+commit-access log an issue or contact a WG member directly and it will
+be brought up in the next WG meeting.
+
+Modifications of the contents of the LTS and Release repository are made 
+on a collaborative basis.  Anybody with a GitHub account may propose a
+modification via pull request and it will be considered by the project
+Collaborators. All pull requests must be reviewed and accepted by a
+Collaborator with sufficient expertise who is able to take full responsibility
+for the change. In the case of pull requests proposed by an existing
+Collaborator, an additional Collaborator is required for sign-off. Consensus
+should be sought if additional Collaborators participate and there is
+disagreement around a particular modification. See Consensus Seeking
+Process below for further detail on the consensus model used for governance.
+
+Collaborators may opt to elevate significant or controversial modifications,
+or modifications that have not found consensus to the WG for discussion by
+assigning the WG-agenda tag to a pull request or issue. The WG should serve
+as the final arbiter where required.
+
+For the current list of Collaborators, see the project README.md.
+
+## WG Membership
+
+WG seats are not time-limited. There is no fixed size of the WG. However,
+the expected target is between 6 and 12, to ensure adequate coverage of
+important areas of expertise, balanced with the ability to make
+decisions efficiently.
+
+There is no specific set of requirements or qualifications
+for WG membership beyond these rules.
+
+The WG may add additional members to the WG by unanimous consensus.
+
+A WG member may be removed from the WG by voluntary resignation,
+or by unanimous consensus of all other WG members.
+
+Changes to WG membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "WG Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full WG
+is not in attendance to participate, then the addition or removal is
+added to the agenda for the subsequent meeting. This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions. If a WG member is unable to attend a meeting
+where a planned membership decision is being made,
+then their consent is assumed.
+
+No more than 1/3 of the voting WG members may be affiliated with the same
+employer. If removal or resignation of a WG member, or a change of
+employment by a WG member, creates a situation where more than 1/3
+of the WG membership shares an employer, then the situation must be
+immediately remedied by the resignation or removal of one or more
+WG members affiliated with the over-represented employer(s).
+
+## WG Meetings
+
+The WG meets weekly on a Google Hangout On Air. A designated moderator
```

Comment:
I believe the current meeting frequency, at least for the LTS WG, is once every three weeks. I'm not sure about the Release team specifically.